packages/mobile/src/screens/edit-track-screen/EditTrackScreen.tsx

```diff
@@ -0,0 +1,48 @@
+import type { UploadTrack } from '@audius/common'
+import { Formik } from 'formik'
+import * as Yup from 'yup'
+
+import { EditTrackNavigator } from './EditTrackNavigator'
+import type { FormValues, EditTrackScreenProps } from './types'
+
+const EditTrackSchema = Yup.object().shape({
+  title: Yup.string().required('Required'),
+  artwork: Yup.object({
```

Comment:
so if a track's artwork already exists, we want to leave "artwork" null so that it doesn't upload the existing artwork again. was following the desktop pattern here